easy_profiler_core/CMakeLists.txt

```diff
@@ -62,6 +63,21 @@ set(BUILD_WITH_CHRONO_HIGH_RESOLUTION_CLOCK OFF CACHE BOOL "Use std::chrono::hig
 
 
 
+#####################################################################
+# Generate config header:
+if (${EASY_OPTION_THREAD_SAFE_INIT})
+    set(EASY_DISABLE_THREAD_SAFE_INIT 0)
+else ()
+    set(EASY_DISABLE_THREAD_SAFE_INIT 1)
+endif ()
+
+configure_file (
```

Comment:
Could you, please, use existing `easy_define_target_option` cmake command instead of generating new file?